CONTRIBUTING.md

```diff
@@ -0,0 +1,151 @@
+issue tracker, pull requests, and IRC channel, is expected to treat
+other people with respect and more generally to follow the guidelines
+articulated in the [Python Community Code of
+Conduct](https://www.python.org/psf/codeofconduct/).
+
+
+Submitting Changes
+------------------
+
+Even more excellent than a good bug report is a fix for a bug, or the
+implementation of a much-needed new feature.  We'd love to have your
+contributions.
+
+We use the usual GitHub pull-request flow, which may be familiar to
+you if you've contributed to other projects on GitHub.  For the mechanics
+of it, see [our git and GitHub workflow help page](http://www.mypy-lang.org/wiki/UsingGitAndGitHub),
+or [GitHub's own documentation](https://help.github.com/articles/using-pull-requests/).
+
+Anyone interested in Mypy may review your code.  One of the Mypy core
+developers will merge your pull request when they think it's ready.
+For every pull request, we aim to promptly either merge it or say why
+it's not yet ready; if you go a few days without a reply, please feel
+free to ping the thread with a new comment.
+
+At present the core developers are (alphabetically):
+* David Fisher (@ddfisher)
+* Jukka Lehtosalo (@JukkaL)
+* Greg Price (@gnprice)
+* Guido van Rossum (@gvanrossum)
+
+
+Preparing Changes
+-----------------
+
+Before you begin: if your change will be a significant amount of work
+to write, we highly recommend starting by opening an issue laying out
+what you want to do.  (This is good advice for all kinds of
+open-source projects in general.)  That lets a conversation happen
+early in case other contributors disagree with what you'd like to do
+or have ideas that will help you do it.
+
+The best pull requests are focused, clearly describe what they're for
+and why they're correct, and contain tests for whatever changes they
+make to the code's behavior.  As a bonus these are easiest for someone
+to review, which helps your pull request get merged quickly!  Standard
+advice about good pull requests for open-source projects applies; we
+have [our own writeup](http://www.mypy-lang.org/wiki/GoodPullRequest)
+of this advice.
+
+See also our [coding conventions](http://www.mypy-lang.org/wiki/CodeConventions) --
+which consist mainly of a reference to
+[PEP 8](https://www.python.org/dev/peps/pep-0008/) -- for the code you
+put in the pull request.
+
+You may also find other pages in the
+[Mypy developer guide](http://www.mypy-lang.org/wiki/DeveloperGuides)
+helpful in developing your change.
+
+
+Issue-tracker conventions
+-------------------------
+
+We aim to reply to all new issues promptly.  We'll assign a
+"milestone" to help us track which issues we intend to get to when,
+and may apply "labels" to carry some other information.  Here's what
+our milestones and labels mean.
+
+### Milestones
+
+We use GitHub "milestones" (see [our
+milestones](https://github.com/python/mypy/milestones)) to roughly
+order what we want to do soon and less soon.
```

Comment:
Less is an adjective according to Merriam Webster, so "less soon" doesn't quite make sense. Maybe "not-so-soon"?